nums/numpy/__init__.py

```diff
@@ -31,10 +34,39 @@ def _init():
     # pylint: disable=import-outside-toplevel
     import numpy as np
     from nums.core.systems import utils as system_utils
+    failed_fallback = settings.failed_fallback
     for name, func in system_utils.get_module_functions(np).items():
         if name not in globals():
-            globals()[name] = _not_implemented(func)
+            # TODO(mwe): Allow failed fallback functions to be used in default function doctests
+            if hasattr(np, func.__name__) and func.__name__ not in failed_fallback:
+                globals()[name] = _default_to_numpy(func)
+            else:
+                globals()[name] = _not_implemented(func)
+            if hasattr(np, func.__name__):
+                globals()[name].__doc__ = update_doc_string(np.__getattribute__(func.__name__)
+                                                            .__doc__)
 
 
+def update_doc_string(doc_string):
```

Comment:
Is there an appropriate utils module we can move this function to?

Also, if the generated doc string is just for running tests, then can we set the docs using this function in tests and not in production code? Would be great to have updated docs in production code, and I think at this point just providing the numpy doc string as-is is probably ok.

Reply:
I think this is a good idea, yes.